macro/Version.py

```diff
@@ -47,11 +47,10 @@
 def execute(macro, args):
     if args:
         version = str(args)
-        if version.lower() == 'm-turtle':
-            # TODO Change when m-turtle name is decided
+        if version.lower() == 'melodic':
             return ('<span style="background-color:#FFFF00; '
                     'font-weight:bold; padding: 3px;">'
-                    'Expected in M-Turtle</span>')
+                    'Expected in Melodic Morenia</span>')
```

Comment:
Yeah arguably we could also make the logic different in the future to make sure Version is passed strings that are actually known distros to prevent "New in TacoTaco"